templates/email/bexley/waste/_sidebar_content.html

```diff
@@ -7,6 +8,7 @@
   ELSIF is_missed_collection;
     title = report.title;
     title = title.remove('Report missed ');
+    title = title _ ' (' _ containers.$title _ ')';
```

Comment:
This feels a bit fragile, to rely upon a title text string lookup. I may be missing something here, but if "Report missed" is being removed from the title, why are we adding it in the first place in `Bexley/Waste.pm`? Could we not add it when we construct the title at report creation, also add the type of container to the title there, and then we don't need to add it here or remove any text. I don't think anything expects the title to be of a particular format.

Reply:
I've not checked whether there is any other objection to not adding 'Report missed' to the service, but does seem to make sense...